packages/nc-gui-v2/composables/useLTARStore.ts

```diff
@@ -0,0 +1,257 @@
+import type { ColumnType, LinkToAnotherRecordType, PaginatedType, TableType } from 'nocodb-sdk'
+import type { Ref } from 'vue'
+import { Modal, notification } from 'ant-design-vue'
+import { useInjectionState, useMetas, useProject } from '#imports'
+import { NOCO } from '~/lib'
+import type { Row } from '~/composables'
+import { extractSdkResponseErrorMsg } from '~/utils'
+
+interface DataApiResponse {
+  list: Record<string, any>
+  pageInfo: PaginatedType
+}
+
+const [useProvideLTARStore, useLTARStore] = useInjectionState(
```

Comment:
A comment what `LTAR` even means would be nice so others understand the name as well (probably Link to another Record, right?)